.github/workflows/cloudflare-preview.yml

```
@@ -1,101 +1,2 @@
-                body: commentBody
-              });
-            } else {
-              // Create new comment
-              await github.rest.issues.createComment({
-                owner: context.repo.owner,
-                repo: context.repo.repo,
-                issue_number: prNumber,
-                body: commentBody
-              });
-            }
+# Replaced by build-pr-preview.yml + deploy-pr-preview.yml
```

actionlint (1.7.11)

[error] workflow is empty

(syntax-check)